shared/middleware/net_sim_api.rb

```diff
@@ -0,0 +1,111 @@
+require 'sinatra/base'
+require 'cdo/db'
+require 'cdo/rack/request'
+require 'csv'
+
+class NetSimApi < Sinatra::Base
+
+  helpers do
+    [
+        'core.rb',
+        'storage_id.rb',
+        'table.rb',
+    ].each do |file|
+      load(CDO.dir('shared', 'middleware', 'helpers', file))
+    end
+  end
+
+  TableType = CDO.use_dynamo_tables ? DynamoTable : Table
+
+  # Environment-dependent channel ID for NetSim.
+  # Will no longer be needed when we detach from the channels API entirely.
+  def netsim_channel_id
+    rack_env?(:development) ? 'JGW2rHUp_UCMW_fQmRf6iQ==' : 'HQJ8GCCMGP7Yh8MrtDusIA=='
```

Comment:
Would be better to move this into the config. (That way each environment's config can provide the appropriate value rather than making this code choose the key.)